src/app/public/modules/card/card.component.html

```diff
@@ -7,6 +7,9 @@
     'sky-card-selected': selectable &amp;&amp; selected
   }"
 >
+  <ng-content
+    select="sky-inline-delete"
+  ></ng-content>
```

Comment:
The "Delete" and "Cancel" buttons seem to be positioned slightly higher than the prototype. Maybe more of a question for @Blackbaud-ToddRoberts . Is that a problem?

<img width="382" alt="Screen Shot 2020-01-09 at 1 35 09 PM" src="https://user-images.githubusercontent.com/18354182/72094665-decb0200-32e4-11ea-93de-2875d4c4c05c.png">

![Screen Shot 2020-01-09 at 1 35 24 PM](https://user-images.githubusercontent.com/18354182/72094686-eee2e180-32e4-11ea-8991-45095f74fc14.png)

Reply:
looks like it's 10px which is correct

Reply:
Sounds good to me then!